### PR TITLE
docs: Add wrapper script for container installs

### PR DESCRIPTION
**Pull request by PhrozenByte**
Follow-up to https://github.com/coreos/butane/pull/294

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -38,6 +38,19 @@ alias butane='podman run --rm --tty --interactive \
               quay.io/coreos/butane:release'
 ```
 
+Alternatively you could also create a wrapper script at `~/.local/bin/butane`:
```
Comment (travier) [inline docs/getting-started.md:41]:
```suggestion
Alternatively you may also create a wrapper script at `~/.local/bin/butane`:
```

Minor, to keep the wording consistent with the previous sentences.

File: docs/getting-started.md [hunk 2]
```diff
@@ -38,6 +38,19 @@ alias butane='podman run --rm --tty --interactive \
...
+
+```bash
+#!/bin/sh
+exec podman run --rm --tty --interactive \
+     --security-opt label=disable        \
+     --volume "$PWD":/pwd --workdir /pwd \
+     quay.io/coreos/butane:release       \
+     "$@"
```
Comment (travier) [inline docs/getting-started.md:49]:
```suggestion
     --volume "${PWD}":/pwd --workdir /pwd \
     quay.io/coreos/butane:release       \
     "${@}"
```

Also minor. I'm wondering why we do not have quotes for the $PWD entry right above in the docs.

**Review by travier — APPROVED** [review]
LGTM. Minor tweaks only.

**Comment by PhrozenByte** [comment]
All fixed @travier :+1: Additionally I quoted `$PWD` in the alias.

**Review by travier — APPROVED** [review]

**Comment by PhrozenByte** [comment]
There still was `--tty` in my script :facepalm: Luckily I checked again :sweat_smile:

**Review by travier — APPROVED** [review]

**Review by bgilbert — APPROVED** [review]
Thanks!

**Pull request merged**